io.github.jkozera.ZevDocs.json

```diff
@@ -0,0 +1,146 @@
+{
+    "app-id" : "io.github.jkozera.ZevDocs",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.golang"
+    ],
+    "command" : "zevdocs",
+    "rename-icon": "zevdocs",
```

Comment:
You should probably use `copy-icon` too. Otherwise ZevDocs might not have proper icon displayed on taskbar/in-app.

Reply:
Hmm. I've patched the sources of ZevDocs to refer to `io.github.jkozera.ZevDocs` icon instead of `zevdocs`, specifically to make it work, and it does for me. :) (See https://github.com/jkozera/zevdocs/blob/17f79dfbff8f2c8a90be4c7431ba5d2ed91c3c8c/src/dh-app.c#L498 for instance)

Reply:
Oh, scratch that, actually it was broken in About dialog and your idea fixes it. I'll update it. And I'll also fix screenshot, since that's broken too. :)

Reply:
Done. :)